src/prefect/transactions.py

```diff
@@ -203,6 +212,51 @@ def __enter__(self) -> Self:
 
         # this needs to go before begin, which could set the state to committed
         self.state = TransactionState.ACTIVE
+
+    def add_child(self, transaction: Self) -> None:
+        self.children.append(transaction)
+
+    def get_parent(self) -> Self | None:
+        parent = None
+        if self._token:
+            prev_var = getattr(self._token, "old_value")
```

Comment:
curious about this `getattr` here

Reply:
it might have been overly defensive code from when this was first wired up, i think all tokens have this attribute afaik